src/netius/test/extra/proxy_c.py

```
@@ -708,6 +751,81 @@ def test_apply_tags_error_url_empty(self):
 
         self.assertTrue("myapp" not in self.server.error_urls)
 
+    def test_apply_tags_redirect(self):
+        tags = ["proxy.enable=true", "proxy.redirect=other.host.com"]
+        entries = [("myapp", "myapp", ["http://10.0.0.1:8080"], tags)]
+        self.server._build_hosts(entries)
+
+        self.assertEqual(self.server.redirect.get("myapp"), "other.host.com")
```

Pylint (4.0.5)

[error] Instance of 'ConsulProxyServer' has no 'redirect' member

(E1101)